systemtests/src/test/java/org/bf2/admin/kafka/systemtest/plain/RecordEndpointTestIT.java

```diff
@@ -229,23 +230,24 @@ void testConsumeRecordsByStartingOffset(int startingOffset, int expectedResults)
 
     @ParameterizedTest
     @CsvSource({
-        "1",
-        "2",
-        "3",
-        "4",
-        "20"
+        "10",
+        "20",
+        "30",
+        "40",
+        "200"
     })
     void testConsumeLatestRecords(int limit) {
```

Comment:
Looks like the issue is with the fixed `partition` parameter in the fetch. Try removing `.queryParam("partition", 0)` further down in this test case.